src/Validator/Rules/UniqueInputFieldNames.php

```diff
@@ -54,7 +55,8 @@ public function getASTVisitor(ValidationContext $context): array
                     $this->knownNames = $knownNames;
                 },
             ],
-            NodeKind::OBJECT_FIELD => function (ObjectFieldNode $node) use ($context): VisitorOperation {
+            NodeKind::OBJECT_FIELD => function ($node) use ($context): VisitorOperation {
```

Comment:
```suggestion
            NodeKind::OBJECT_FIELD => function (mixed $node) use ($context): VisitorOperation {
```

is this already supported in this php version?

Reply:
nah, `mixed` is PHP 8. We could do `Node` or something, but I think just the `assert()` is cleanest.